monkestation/code/modules/replays/subsystem/replay.dm

```diff
@@ -495,4 +537,4 @@ SUBSYSTEM_DEF(demo)
 	if(marked_dirty[M])
 		marked_dirty -= M
 	if(initialized)
-		del_list["\ref[M]"] = 1
+		del_list[ref(M)] = TRUE
```

Comment:
i benchmarked it - lone `ref(thing)` is faster than lone `"\ref[thing]"` - but only by itself, `\ref[thing]` is faster when part of a larger string